packages/core/src/createPrepareTransfers.ts

```diff
@@ -83,63 +83,127 @@ export interface PrepareTransfersProps {
 }
 
 /**
- * Create a [`prepareTransfers`]{@link #module_core.prepareTransfers} function by passing an optional network `provider`.
- * It is possible to prepare and sign transactions offline, by omitting the provider option.
  *
  * @method createPrepareTransfers
- *
+ * 
+ * @summary Creates a new `prepareTransfers()` method.
+ * 
  * @memberof module:core
  *
- * @param {Provider} [provider] - Optional network provider to fetch inputs and remainder address.
- * In case this is omitted, proper input objects and remainder should be passed
- * to [`prepareTransfers`]{@link #module_core.prepareTransfers}, if required.
+ * @param {Provider} [provider] - Optional provider object that the method should use to call the node's API endpoints.
+ * To create transactions offline, omit this parameter so that the returned function does not get your addresses and balances from the node. To create value transactions offline, make sure to pass input objects and a remainder address to the returned function.
+ * 
+ * @example
+ * ```js
+ * const prepareTransfersOffline = Iota.createPrepareTransfers();
+ * 
+ * const transfers = [
+ *  {
+ *    value: 1,
+ *    address: 'RECEIVINGADDRESS...'
+ *  }
+ * ];
+ * 
+ * prepareTransfersOffline(seed, transfers, {
```

Comment:
Why are calling it `prepareTransfersOffline` instead of just `prepareTransfers`?